tests/conftest.py

```diff
@@ -17,6 +17,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import pytest
+import splinter
 from dotenv import find_dotenv, load_dotenv
 
 load_dotenv(find_dotenv())
+
+
+@pytest.fixture()
+def browser():
```

Comment:
Do you plan to use it in this PR?

Reply:
it's used in the test. I was using the pytest-splinter extension before but running into problems so I defined my own browser fixture here. The name stays the same.